lv_cmsis_pack.c

```diff
@@ -0,0 +1,115 @@
+/****************************************************************************
+*  Copyright 2022 Gorgon Meducer (Email:embedded_zhuoran@hotmail.com)       *
+*                                                                           *
+*  Licensed under the Apache License, Version 2.0 (the "License");          *
+*  you may not use this file except in compliance with the License.         *
+*  You may obtain a copy of the License at                                  *
+*                                                                           *
+*     http://www.apache.org/licenses/LICENSE-2.0                            *
+*                                                                           *
+*  Unless required by applicable law or agreed to in writing, software      *
+*  distributed under the License is distributed on an "AS IS" BASIS,        *
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+*  See the License for the specific language governing permissions and      *
+*  limitations under the License.                                           *
+*                                                                           *
+****************************************************************************/
+
+/**
+ * @file lv_anim.c
+ *
+ * @brief This file will only be used by cmsis-pack.
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "RTE_Components.h"
```

Comment:
Please add `#if` guards to avoid automatically picking this file.

Reply:
Sorry, I don't understand. Who will automatically pick this file?
It is only be used by cmsis-pack.

Reply:
E.g. I use Eclipse and it tries to compile all the C files by default. Of course you can exclude files or directories but now LVGL is configuration free in this regards and it would be great to keep it.

Reply:
Emm, OK, let me fix this. I will 
1) change its name from lv_cmsis_pack.c to lv_cmsis_pack.txt
2) use gen_pack.sh to rename it as lv_cmsis_pack.c when creating packs. 

What do you think?

Reply:
Done.